### PR TITLE
pantheon -> besu

### PR DESCRIPTION
**Pull request by joshuafernandes**
Signed-off-by: Joshua Fernandes <joshua.fernandes@consensys.net>

### CODE REVIEW

File: helmfile/private-network-ibft-automated/besu-node/Chart.yaml [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 appVersion: 1.1.4
 keywords:
   - validator
   - bootnode
   - node
+  - besu
+  - hyperledger
   - pantheon
   - ethereum
   - pantheon
```
Comment (EdJoJob) [inline helmfile/private-network-ibft-automated/besu-node/Chart.yaml]:
```suggestion

```

File: helm/private-network-ibft/besu/README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ node:
 
 #### 4. Run helm and install the chart
 ```bash
-helm install --namespace NAMESPACE --name pantheon ./pantheon
+helm install --namespace NAMESPACE --name besu ./besu
 ```
 
 #### 5. In the dashboard, you will see each bootnode deployment & service, nodes & a node service, miner if enabled, secrets(opaque) and a configmap
```
Comment (EdJoJob) [inline helm/private-network-ibft/besu/README.md]: Line 61 there is a `PANTHEON_NODE_SERVICE_HOST` that was missed

Comment (joshuafernandes) [reply helm/private-network-ibft/besu/README.md]: fixed

File: helmfile/private-network-ibft-automated/besu-node/Chart.yaml (hunk 1 above)
Comment (joshuafernandes) [reply helmfile/private-network-ibft-automated/besu-node/Chart.yaml]: I've removed `pantheon` - having pinged Arnaud to have a look at the helmfile as well

**Pull request merged**